doc/getting-started/README.md

```diff
@@ -1,26 +1,27 @@
 # Getting started
 
-The `CassandraCSharpDriver.Graph` package leverages the features of [Gremlin.Net language variant][glv] and the high-level client driver
-features of the [DataStax C# Driver for Apache Cassandra][driver].
+The `CassandraCSharpDriver.Graph` package leverages the features of [Gremlin.Net language variant][glv] and the high-level client driver features of the [DataStax C# Driver for Apache Cassandra][driver].
+
+This package provides the `fluent` API and the core driver package (`CassandraCSharpDriver`) provides the `script` API. For more information about the `script` API, see [the `Graph Support` section of the core driver documentation][graph-support]. That section of the core driver documentation covers some topics that are not covered here so it's recommended to read it as well.
```

Comment:
I think `fluent` and `script` API can be confusing for users and bystanders.

It could be something like: 

```
This package provides the fluent, builder-like API for graph traversal execution while the core driver package supports gremlin traversal string execution. [...]
```

Reply:
Hmm I should probably change https://github.com/datastax/csharp-driver/pull/543 as well then because my changes there also reference a `script` API

Reply:
done